dependencies.Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Dependabot can keep this file up to date with latest containers.
 
 # Weaver is used to generate markdown docs, and enforce policies on the model.
-FROM otel/weaver:0.8.0 AS weaver
+FROM otel/weaver:v0.9.0 AS weaver
```

Comment:
I noticed that the image tagging scheme was indeed changed as can be seen at https://hub.docker.com/r/otel/weaver/tags but was this intentional or will this be reverted?

Reply:
This is intentional.  The docker provided github actions provide a best-practice versioning scheme and we now match that.

- Versions now match the tag in github
- `latest` means latest release
- `main` matches the branch in github.

Features would do the same.